app.py

```diff
@@ -49,7 +56,47 @@ def asynchronous_push(f_stop):
 asynchronous_push(f_stop)
 
 # Now let's run the app!
-chatbot = pipeline(model="microsoft/DialoGPT-medium")
+prompt = load_prompt(PROMPT_TEMPLATES / "openai_chatgpt.json")
+
+chatbot_1 = ConversationChain(
```

Comment:
For now this is hardcoded, but we could simplify by looping over a list of model IDs and then creating a list of `ConversationalChain` objects